sonar-maven-plugin/src/main/java/org/sonarsource/scanner/maven/bootstrap/MavenUtils.java

```diff
@@ -94,6 +97,34 @@ static void putAll(Properties src, Map<String, String> dest) {
     }
   }
 
+  /**
+   * Copy all items, except irrelevant encrypted properties, that is,
+   * entries which are encrypted and whose name does not contain "sonar".
+   */
+  static void putRelevant(Properties src, Properties dest) {
+    putRelevant(src, dest::put);
+  }
+
+  /**
+   * See {@link #putRelevant(Properties, Properties)}.
+   */
+  static void putRelevant(Properties src, Map<String, String> dest) {
+    putRelevant(src, dest::put);
+  }
+
+  // Matches strings starting with "{AES}", "{b64}", "{whatever}", etc.
+  private static final Pattern ENCRYPTED_VAR = Pattern.compile("^\\{[a-zA-Z0-9]*\\}.*");
```

Comment:
Do we want to cap the number of signs between curly brackets? The [implementation ](https://github.com/SonarSource/sonarqube/blob/0cb598a8d618308002ee5c9a15f43b9d13ab2dc3/sonar-plugin-api-impl/src/main/java/org/sonar/api/config/internal/Encryption.java#L66)we work against mentions at least 1 character
```suggestion
  private static final Pattern ENCRYPTED_VAR = Pattern.compile("^\\{[a-zA-Z0-9]++\\}.*");
```

Reply:
Done. I also added `_-` to the pattern, because the file you linked has `aes-gcm` algorith that we didn't match.